python/modules/IcePy/ObjectAdapter.cpp

```diff
@@ -1768,7 +1768,7 @@ IcePy::wrapObjectAdapter(const Ice::ObjectAdapterPtr& adapter)
     {
         return nullptr;
     }
-    PyObject* wrapperType = lookupType("Ice.ObjectAdapterI.ObjectAdapterI");
+    PyObject* wrapperType = lookupType("Ice._ObjectAdapterI.ObjectAdapterI");
```

Comment:
The `_` prefix is used for private modules. I renamed private classes to use this prefix to help exclude them from the generated documentation.